changelogs/fragments/6171-shutdown-using-systemd.yml

```
@@ -0,0 +1,2 @@
+bugfixes:
```

I think this is more like a new feature and not a bugfix (since the module didn't promise it supports systemd before):
```suggestion
minor_changes:
```